javascript/javascript2/week3/timer.js

```diff
@@ -0,0 +1,152 @@
+ //Log out the text Called after 2.5 seconds 2.5 seconds after the script is loaded.
+
+ let timeout = setTimeout(()=>{
+    console.log('Called after 2.5 seconds')},2500);
+
+ //Create a function that takes 2 parameters: delay and stringToLog
+        
+ function delayFunction(delay, stringToLog){
+          setTimeout(() => {
+            console.log(stringToLog)
+          }, delay * 1000);
+      }
+      delayFunction(5,'appear after 5 sec');
+      
+ //Create a button in html. button should work after 5 sec
+
+      const button =document.getElementById('myBtn');
+      button.addEventListener('click',()=>{
+          delayFunction(5,'appear after 5sec clicking the button');
+          });
+
+ // Create two functions and assign them to two different variables.
+   function earth(){
+       const planet1 = " ";
+        console.log('earth');
+   }
+   function saturn(){
+       const palnet2 = " ";
+      console.log('saturn');
+   }
+     const planetLogFunction =(planetLogFunction)=>{
+        planetLogFunction();
+    }
+
+   planetLogFunction(earth);
+   planetLogFunction(saturn);
+
+   //Create a function called runAfterDelay. It has two parameters: delay and callback. 
+   //When called the function should wait delay seconds and then call the provided callback function.
+
+  function runAfterDelay(delay, callback){
+       setTimeout(callback,delay*1000);
+   }
+   runAfterDelay(4, function(){
+       console.log('should be logged after 4 seconds');
+   });
+   runAfterDelay(1, function(){
+    console.log('😀');
+});
+// double click (somehow it is not working)
+ 
+ 
+ let doubleClick = document.getElementById('body');
+ doubleClick.addEventListener('dblclick',function(){
+     console.log('double click');
+ });
+ 
```

Comment:
try changing it to 
`document.addEventListener('dblclick',function(){
     console.log('double click');
 });`